src/GraphQL.Tests/Introspection/IntrospectionResult.cs

```diff
@@ -32,196 +32,6 @@ public class IntrospectionResult
           ""enumValues"": null,
           ""possibleTypes"": null
         },
-        {
```

Comment:
I think we need to keep a test (this one) that includes all scalars, and a new test that includes the filtered ones.

I'm also concerned about removing the built-in scalars from introspection before this is an official part of the spec.  Does the `graphql-js` project do that today?  Does GraphiQL and GraphQL Playground still work with these changes?

Reply:
https://github.com/graphql/graphql-js/issues/2005

Reply:
@sungam3r Can you summarize please?  I don't have time to do the whole investigation.

Reply:
So far I have returned the builtin scalars to introspection. I will be back with an answer when I check graphq-js and GraphiQL.

Reply:
> I'm also concerned about removing the built-in scalars from introspection before this is an official part of the spec. 

Now it is.

It looks like the spec has become clearer with respect to embedded scalars - graphql/graphql-spec#597 :

> When returning the set of types from the `__Schema` introspection type, all
referenced built-in scalars must be included. If a built-in scalar type is not
referenced anywhere in a schema (there is no field, argument, or input field of
that type) then it must not be included.

\+

> When representing a GraphQL schema using the type system definition language,
all built-in scalars must be omitted for brevity.